docs/source/host/start-hosting.rst

```diff
@@ -11,12 +11,12 @@ We are thrilled you've decided to host a custom domain with us on the metaverse!
 Overview of Hosting
 ----------------------------
 
-High Fidelity's hosting interface consists of the following components that work together to create an immersive VR experience for your visitors:
+Project Athena's hosting interface consists of the following components that work together to create an immersive VR experience for your visitors:
 
-* The *Sandbox* is a testing environment hosted locally on your computer that gives you a space to play around and build content without affecting any domains in the metaverse. Note that it is possible to configure High Fidelity so that your Sandbox is also a registered domain that is hosted directly from your local computer. The Sandbox application can be download from the `High Fidelity website <https://www.highfidelity.com/download#sandbox>`_.
-* A *domain* is a registered location in the metaverse. Each domain is assigned a unique ID to identify it. You can sign up for a new domain in one of two ways: either by setting up a High Fidelity server or by logging into your account via the High Fidelity website.
+* The *Sandbox* is a testing environment hosted locally on your computer that gives you a space to play around and build content without affecting any domains in the metaverse. Note that it is possible to configure Project Athena so that your Sandbox is also a registered domain that is hosted directly from your local computer. The Sandbox application can be download from the `Project Athena website <https://www.highfidelity.com/download#sandbox>`_.
+* A *domain* is a registered location in the metaverse. Each domain is assigned a unique ID to identify it. You can sign up for a new domain in one of two ways: either by setting up a Project Athena server or by logging into your account via the Project Athena website.
 * A *server* hosts a domain, and processes incoming network requests for the domain. The server for your domain is either a physical computer or a cloud-hosted server.
-* A *droplet* or *cloud domain* is a cloud server hosted by our partner, DigitalOcean.
+* A *droplet* or *cloud domain* is a cloud server hosted on DigitalOcean.
 * A *content set* is the content that you've designed to make your domain unique and pretty. 
 
 ----------------------------
```

Comment:
Also, in the line ...
:doc:`Choose your hosting platform <start-hosting>` (local server or cloud services)
… the parentheses should perhaps be …
(local server, remote server, or cloud services)